src/DETHRACE/common/utility.c

```diff
@@ -453,7 +477,11 @@ br_pixelmap* DRPixelmapLoad(char* pFile_name) {
     if (the_map != NULL) {
         the_map->origin_x = 0;
         the_map->origin_y = 0;
+#if defined(DETHRACE_FIX_BUGS)
```

Comment:
instead of introducing `align_pixels`, what if we just skipped overriding the row_bytes?

```
#if !defined(DETHRACE_FIX_BUGS)
the_map->row_bytes = (the_map->row_bytes + sizeof(int) - 1) & ~(sizeof(int) - 1);
#endif
```

Reply:
In general, the idea of aligning data is to accelerate access times.
I remember reading some platforms not supporting unaligned access, requiring the operating system to handle these.

Reply:
I applied your suggestion in b9d600b

Looking back at my initial custom `allign_pixelmap` function, I think I also needed to modify the `br_pixelmap`'s `flags` member as follows:

```c
pm->flags &= ~BR_PMF_LINEAR;